panel/chat/feed.py

```diff
@@ -205,7 +205,7 @@ class ChatFeed(ListPanel):
         The placeholder wrapped in a ChatMessage object;
         primarily to prevent recursion error in _update_placeholder.""")
 
-    _callback_future = param.ClassSelector(class_=asyncio.Future, allow_None=True, doc="""
```

Comment:
I don't think it needs to be a parameter in the first place, just make it an attribute.